GroceryStore/GroceryStore.Models/Customer.cs

```diff
@@ -0,0 +1,113 @@
+﻿using GroceryStore.Constants;
+using System.Diagnostics;
+using System.Dynamic;
+using System.Globalization;
+using System.Transactions;
+using System.Xml.Linq;
+
+namespace GroceryStore.Models
+{
+    public class Customer
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public int Age { get; set; }
+        public string Sex { get; set; }
+
+        private bool _hasDiscountCard;
+        public bool HasDiscountCard
+        {
+            get
+            { return _hasDiscountCard; }
+            set
+            { _hasDiscountCard = value; }
+        }
+        public string DiscountCardToString
+        {
```

Comment:
In my opinion it's better to create private method ConvertDiscountCardToString()